src/aiida/cmdline/commands/cmd_presto.py

```diff
@@ -55,8 +56,41 @@ def get_default_presto_profile_name():
     show_default=True,
     help='Email of the default user.',
 )
+@click.option(
```

Comment:
The docstring below probably needs to be updated to reflect the new option.

Reply:
Don't understand this one. What needs to be changed exactly because of what new option?

Reply:
I have reworded the description considerably and incorportated the new feature.